CONTRIBUTING.md

```diff
@@ -1,40 +1,118 @@
+If you have a general or technical question, you can use one of the following resources instead of submitting an issue:
 
-#### **Did you write a patch that fixes a bug?**
+- [**Developer documentation:**](https://docs.iota.org/) For official information about developing with IOTA technology
+- [**Discord:**](https://discord.iota.org/) For real-time chats with the developers and community members
+- [**IOTA cafe:**](https://iota.cafe/) For technical discussions with the Research and Development Department at the IOTA Foundation
+- [**StackExchange:**](https://iota.stackexchange.com/) For technical and troubleshooting questions
 
-* [New issues](https://github.com/iotaledger/iri/issues/new) should be used for reporting issues. If you already wrote a patch for an issue that you or someone else reported, make sure you link it to the open issue. You can do this by mentioning the issue number in the comment or description on the PR, for example `#123` to link issue "123". 
+## Ways to contribute
 
-#### **Do you intend to add a new feature or change an existing one?**
+To contribute to IRI on GitHub, you can:
 
-* Suggest your change as a [new issues](https://github.com/iotaledger/iri/issues/new). Use the second part of the issue template, dedicated to new feature requests and delete the first part. If you plan on implementing the change yourself, start writing the code and submit a PR. Note that the fact that the feature is "code ready" doesn't mean it will get merged. It is advisable to gather feedback for the change first. You are encouraged to start and drive a discussion on [Discord](https://discord.iota.org/). The IRI team also monitors the repo and will provide feedback where relevant.
+- Report a bug
+- Suggest a new feature
+- Build a new feature
+- Contribute to the documentation
 
-#### **Want to write a regression test for your feature? Or to any other feature that needs a test?**
+## Report a bug
 
-* Please refer to our current [regression tests](https://github.com/iotaledger/iri/tree/dev/python-regression).
+This section guides you through reporting a bug. Following these guidelines helps maintainers and the community understand the bug, reproduce the behavior, and find related bugs.
 
+### Before reporting a bug
 
-#### **Do you have questions about the functioning of IRI nodes, the network or anything IOTA related?**
+Please check the following list:
 
-* Use our [Discord](https://discordapp.com/invite/fNGZXvh) to join the discussion and ask questions.
+- **Do not open a GitHub issue for [security vulnerabilities](SECURITY.MD)**, instead, please contact us at [security@iota.org](mailto:security@iota.org).
```

Comment:
question: who checks this email
Do you know @jakubcech?

Reply:
Good point, looks like the Bee repo uses secops@iota.org

Reply:
Andrea should be there

Reply:
Do both exist? And which one is preferred?

Reply:
Probably best answered by @karimodm

Reply:
I'll wait for him before merging

Reply:
security@ should be used. We are reshaping some groups to make things clearer. secops@ is now being aliased to security@